packages/rancher-kiali-server-1.35/generated-changes/overlay/templates/psp.yaml

```diff
@@ -0,0 +1,67 @@
+{{- if .Values.global.rbac.pspEnabled }}
```

Comment:
unnecessary change; see comment on dependency.yaml

Reply:
resolved in most recent PR commits.